sbt-metals/src/main/scala/MetalsPlugin.scala

```diff
@@ -1,91 +1,91 @@
+package scala.meta.sbt
+
 import sbt._
 import sbt.Keys._
 import java.io._
-import sbt.Def
 
 object MetalsPlugin extends AutoPlugin {
   override def trigger = allRequirements
   override def requires = sbt.plugins.JvmPlugin
   object autoImport {
 
     val metalsCompilerConfig =
```

Comment:
Yak shaving. How about calling this `metalsBuildInfo`? To match `.metals/buildinfo`

Reply:
That's strange, I actually did it in 2c61ebc, except the case: `metalsBuildinfo`. I will rename it to `..Info`. I would also suggest to raneme all related things in the server code.